### PR TITLE
Fixes #6985

### LINKED ISSUE #6985: Jack-o-Pumpkin disappearance trick
Step 1: Put a pumpkin in your hand
Step 2: Try to hatchet it into a neat Jack-o-Lantern
Step 3: Get SS13's 2nd item disappearance trick since the pizza box pizza disappearance trick

### PR DESCRIPTION
**Pull request by 9600bauds**

### CODE REVIEW

File: code/modules/reagents/reagent_containers/food/snacks/grown.dm [hunk 1]
```diff
@@ -77,7 +77,7 @@
 
 /obj/item/weapon/reagent_containers/food/snacks/grown/harebell
 	name = "harebell"
-	desc = "\"I'll sweeten thy sad grave: thou shalt not lack the flower that's like thy face, pale primrose, nor the azured hare-bell, like thy veins; no, nor the leaf of eglantine, whom not to slander, out-sweeten�d not thy breath.\""
+	desc = "\"I'll sweeten thy sad grave: thou shalt not lack the flower that's like thy face, pale primrose, nor the azured hare-bell, like thy veins; no, nor the leaf of eglantine, whom not to slander, out-sweeten�d not thy breath.\""
```
Comment (clusterfack) [inline code/modules/reagents/reagent_containers/food/snacks/grown.dm]: Whats with this?

Comment (9600bauds) [inline code/modules/reagents/reagent_containers/food/snacks/grown.dm]: Atom does weird things like these, it also did it with the "Did you know?" PDA string but there's no visible effect in Dreammaker or in-game

Comment (9600bauds) [inline code/modules/reagents/reagent_containers/food/snacks/grown.dm]: Actually hold on, it DOES fuck it up

Comment (clusterfack) [inline code/modules/reagents/reagent_containers/food/snacks/grown.dm]: Picnic

Comment (9600bauds) [inline code/modules/reagents/reagent_containers/food/snacks/grown.dm]: I don't know why this shit keeps happening

**Pull request merged**